test/clj/rems/api/services/test_resources.clj

```diff
@@ -53,6 +59,16 @@
               :archived false}
              (status-flags res-id))))
 
+    (testing "cannot unarchive if license is archived"
+      (resources/update-resource! {:id res-id
+                                   :archived true})
+      (archive-license! true)
+      (is (not (:success (resources/update-resource! {:id res-id
+                                                      :archived false}))))
+      (archive-license! false)
+      (is (:success (resources/update-resource! {:id res-id
+                                                 :archived false}))))
+
```

Comment:
consider adding a case to `rems.api.test-administration/test-archiving-disabling` also?

Reply:
Done.